idb/dummy.go

```diff
@@ -503,6 +492,12 @@ type AlgoUpdate struct {
 	Closed bool
 }
 
+// Encodes an update or remove operation on an account_data json key.
```

Comment:
The linter is complaining because it wants this comment to start with `AccountDataUpdate`